### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by VathsalaAchar**
Updated existing readme with team information for Phase 4 and 5. Also added documentation for setup. Happy to make changes if I've missed anything.

### CODE REVIEW

**Comment by VathsalaAchar** [comment]
I'm configuring Firebase emulators and updating the setup documentation with this information as well. Will mark as ready for review once that is done.

File: docs/setup.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+VITE_PROJECT_ID="some value"
+VITE_STORAGE_BUCKET="some value"
+VITE_MESSAGING_SENDER_ID="some value"
+VITE_APP_ID="some value"
+VITE_MEASUREMENT_ID="some value"
+```
+
+Now run the following to have the project running locally on `http://localhost:5173/`
+```
+npm run build
```
Comment (AlanCLN) [inline docs/setup.md]: Probably don't need to run 'npm run build' to run locally for development.

Comment (VathsalaAchar) [reply docs/setup.md]: Thanks, it's updated now

**Comment by VathsalaAchar** [comment]
I've added a new doc [docs/login.md](https://github.com/LetsRuminate/ruminate-feedback/blob/faf7d542e7b38d30698184749b7a5b72ae297ba7/docs/login.md)  with steps to set up an admin user on the emulator and login. Do let me know if you can't get it to work or need help.

**Review by Gigi-dev21 — APPROVED** [review]

**Pull request merged**